compatibility.bs

```diff
@@ -1055,11 +1057,61 @@ On mobile platforms, including smaller iPad form factors
   </tbody>
 </table>
 
+
+<h2 id="removal">Deprecation and Removal</h2>
```

Comment:
Thank you for spinning this up. This is a great starting point to at least centralize things. One thing that I want to be clear on is that this will cover more than just HTML impacting changes correct?

Reply:
correct.